chapter-1-basics.md

```diff
@@ -293,12 +308,15 @@ another *sequence* or start a whole new *session* by signaling `ReadyToWork`.
 ![Activities, Sequences and Sessions](./chapter-1/SessionsAndSequences.png)
 
 Now, you will convert theory into practice and begin with starting a *session*.
-To do so, update the `ControlSystemAttached()` method to the following: 
+To do so, the `ControlSystemAttached()` method contains to the following code: 
```

Comment:
I think the phrasing was fine (now grammar is wrong). Maybe something like `as follows` would be a bit better.